db/migrate/20220811231507_add_name_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddNameToUser < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :name, :string
+  end
```

Comment:
I suggest you make the name field a unique index for each user since that would be the only field for user authentication as stated in the requirement. Thank you